### DISCUSSION TITLE
Welcome to OCT-Converter Discussions!

### DISCUSSION THREAD

**Discussion by marksgraham** [marksgraham/OCT-Converter#15] [General]
## Welcome!
  We’re using Discussions as a place to discuss OCT-Converter, without necessarily needing to raise issues. You can:
  * Ask questions you’re wondering about.
  * Share ideas.

**Comment by hg332** [comment]
Hello! Really general question; What is the difference between the Topcon .fds and .fda data type?

**Reply by peteratBHVI** [reply]
I just can report from observations. fds files are much bigger approx 200MB and I think they are an older file format and generated first before compressed into fda.

**Reply by hg332** [reply]
That would mean that there could be OCT and Fundus pictures saved in both .fds and .fda?

**Reply by peteratBHVI** [reply]
I would assume so, but I can not tell fore sure. It might depend on the recording option. fundus photos can be opted out in the settings. That is helpful if you want to take several scans and not want to dazzle participants.

**Reply by marksgraham** [reply]
They can both store OCT and fundus images - I'm not sure why the two formats exist. Perhaps as Peter says the .fda is just a compressed form of the .fds

**Comment by micceo** [comment]
Hi! I'm no way versed in either python or any of the formats here.
Looking into converting e2e to dcm. This project make that possible? Problem I got is that dcm export from Heyex2 (Heidelberg) is missing information (this is an Eye exam instrument and the missing info is of which eye the information is coming, right or left). The information is probably in the e2e, so looking for to convert these before handing them over to a Zeiss Forum server.

**Reply by marksgraham** [reply]
Hey,

So yeah, you should be able to import the .e2e and save as a Dicom. There isnt' direct support for saving dicom's yet in the code but should be able to create a save a dicom with just a few lines of code. It won't have all the dicom tags populated but you should be able to look in and view the volume. The laterality is extracted from .e2e so you could either save that in the filename or we could work out where to store that in the appropriate dicom tag.
